resources/rules/post_identifier_merge/ice_to_bio/uniprot_protein_entrez_gene_mapping.clj

```diff
@@ -0,0 +1,32 @@
+;; -------------------------------------------------------
+;; --------- Uniprot Protein Entrez Gene Mapping ---------
+;; -------------------------------------------------------
+`{:name "uniprot-protein-entrez-gene-mapping"
+   :description "This rule maps uniprot proteins to entrez genes"
+   :head (
+          ;; map proteins to genes
+          (?/protein kso/has_indirect_template ?/gene)
```

Comment:
This is a "shortcut" relation. It's redundant with the restriction defined below. We should decide on a consistent approach to these. There may be a few more scattered throughout the rules. My inclination is to delete them.